flight/targets/quanton/fw/pios_board.c

```diff
@@ -624,7 +624,7 @@ void PIOS_Board_Init(void) {
 		}
 
 		if (PIOS_I2C_CheckClear(pios_i2c_usart1_adapter_id) != 0)
-			panic(6);
```

Comment:
I looked at a GPS/mag unit that I have this problem with, and it has pull up resistors on SCL and SDA.  They effectively become pull downs if the device is connected, but unpowered.